dtfs-central-api/api-tests/v1/user/user-get.api-test.js

```diff
@@ -4,7 +4,7 @@ const { DB_COLLECTIONS } = require('../../../src/constants');
 const api = require('../../api')(app);
 
 const newUser = {
-  username: 'BANK1_MAKER1',
+  username: 'maker1@ukexportfinance.gov.uk',
```

Comment:
Where applicable, I've replaced these with constants -- I've raised another ticket for the tech debt around our user objects in test cases